Add instructions for excluding interfaces in Netifyd (#136)
NethServer/nethsecurity-docs: dpi_filter.rst

Enter the ``IP address`` that should be exempted from the filter.
You can include a description explaining the reason for the exclusion.

Each exception can be enabled or disabled as desired.

Netify interface exclusion
--------------------------

By default, Netifyd monitors all interfaces. To exclude specific interfaces, you can define an exclusion list. Below are commands to add, modify, or remove excluded interfaces.

> - Add interfaces to exclusion list: ::

Reviewer: ```suggestion
- Add interfaces to exclusion list. The system will exclude all interfaces with a name starting with the configure value: ::
```

Enter the ``IP address`` that should be exempted from the filter.
You can include a description explaining the reason for the exclusion.

Each exception can be enabled or disabled as desired.

Netify interface exclusion
--------------------------

> By default, Netifyd monitors all interfaces. To exclude specific interfaces, you can define an exclusion list. Below are commands to add, modify, or remove excluded interfaces.

Reviewer: ```suggestion
By default, Netifyd monitors all interfaces. To exclude specific interfaces, you can define an exclusion list. Below are commands to add, modify, or remove excluded interfaces.
The exclusion list is configured using the ``ns_exclude`` option that takes a list of patterns. Each entry is a shel glob pattern.

> - Add interfaces to exclusion list. The system will exclude all interfaces with a name starting with the configure value: ::

Reviewer: >  The system will exclude all interfaces with a name starting with the configure value

this is not true anymore, explain a bit the globbing

Enter the ``IP address`` that should be exempted from the filter.
You can include a description explaining the reason for the exclusion.

Each exception can be enabled or disabled as desired.

Netify interface exclusion
--------------------------

By default, Netifyd monitors all interfaces. To exclude specific interfaces, you can define an exclusion list. Below are commands to add, modify, or remove excluded interfaces.
The exclusion list is configured using the ``ns_exclude`` option that takes a list of patterns. Each entry is a shel glob pattern.

> - Add interfaces to exclusion list. The system will exclude all interfaces with a name starting with the configure value: ::

Reviewer: ```suggestion
- Add interfaces to exclusion list. The system will exclude the `eth1` interface and all OpenVPN and WireGuard interfaces: ::
```
